tests/Integration/Foundation/FoundationHelpersTest.php

```diff
@@ -144,6 +145,40 @@ public function testFakeUsesLocale()
         $this->assertSame('Australian Capital Territory', fake()->state());
     }
 
+    public function testStatusCodeResponsesWithIntegers()
+    {
+        $code = Arr::random([100, 200, 201, 400, 409, 422, 500, 503]);
```

Comment:
If you want to check multiple values, use a data provider. There's no benefit in catching a bad case on one out of 8 runs on average...